### PR TITLE
feat: Added docs

### PR DESCRIPTION
**Pull request by Murtatrxx**

### CODE REVIEW

**Comment by jacoobes** [comment]
Make it a draft, not finished

**Comment by EvolutionX-10** [comment]
damn the failing checks

File: _blog/authors.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+jacoobes:
```
Comment (Murtatrxx) [inline _blog/authors.yml:1]: why i'm not here 😭

File: docs/guide/walkthrough/plugins.md [hunk 2]
```diff
@@ -1,9 +1,15 @@
...
+sidebar_position: 5
+---
+
 # Plugins
 
-As of now, modules seem a little underwhelming. It appears that sern doesn't have all the features of a standard handler,
-which manages permissions, categorizes, cool-downs, publishes application commands, role permissions, etc. Many important
-parts that manage access and help streamline command creation to make are apparently absent.  
-Below is an example of an event plugin, one of the types of plugins.
+<p>As of now, modules seem a little underwhelming. It appears that sern doesn't have all the features of a standard handler,
```
Comment (Murtatrxx) [inline docs/guide/walkthrough/plugins.md:7]: do we need <p> tags??

File: _blog/authors.yml (hunk 1 above)
Comment (jacoobes) [reply _blog/authors.yml:1]: ah thats only for the blog, which is not done. if you wanna write blogs, you would add yourself here

**Review by jacoobes — APPROVED** [review]

**Pull request merged**